crates/futures/tests/tests.rs

```diff
@@ -1,57 +1,38 @@
 #![cfg(target_arch = "wasm32")]
 
-extern crate futures;
-extern crate js_sys;
-extern crate wasm_bindgen;
-extern crate wasm_bindgen_futures;
-extern crate wasm_bindgen_test;
+// wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
```

Comment:
Was this meant to be kept commented out?

Reply:
Oops no, good catch!